llvm/lib/Target/X86/X86.h

```diff
@@ -199,6 +199,7 @@ void initializeX86ReturnThunksPass(PassRegistry &);
 void initializeX86SpeculativeExecutionSideEffectSuppressionPass(PassRegistry &);
 void initializeX86SpeculativeLoadHardeningPassPass(PassRegistry &);
 void initializeX86TileConfigPass(PassRegistry &);
+void initializeX86IndirectBranchTrackingPassPass(PassRegistry &);
```

Comment:
Is the change related to jump tables? Put it in a separate patch if not.

Reply:
I added that so that it could be run via `-run-pass=x86-ibt`, as required in the test I added (indirect-branch-tracking-jt.mir). I can submit separate patch for it, but this patch will then be dependent on that one.

Reply:
Actually, I rewrote the test at the IR-level so running the x86-ibt pass via `-run-pass=x86-ibt` is no longer necessary. Thus, the above change (as well as `INITIALIZE_PASS(X86IndirectBranchTracking, ...)`), etc.) is no longer necessary, so I've reverted those.